packages/lang-server/src/location.ts

```diff
@@ -28,6 +28,16 @@ export interface SaltoElemLocation {
 
 const MAX_LOCATION_SEARCH_RESULT = 20
 
+const getAllElements = async (workspace: EditorWorkspace):
+Promise<ReadonlyArray<Element>> => {
+  const topLevelElements = await workspace.elements
+  const fields = topLevelElements
+    .filter(isObjectType)
+    .map(elem => Object.values(elem.fields))
+    .flat()
+  return [...topLevelElements, ...fields]
```

Comment:
nit: this can be done in a single pass to improve performance a bit
```suggestion
  return (await workspace.elements)
    .flatMap(elem => isObjectType(elem) ? [elem, ...Object.values(elem.fields)] : [elem])
```